src/providers/ldap/ldap_child.c

```diff
@@ -298,21 +301,21 @@ static krb5_error_code ldap_child_get_tgt_sync(TALLOC_CTX *memctx,
     if (!realm_str) {
         krberr = krb5_get_default_realm(context, &default_realm);
         if (krberr) {
-            DEBUG(SSSDBG_OP_FAILURE, "Failed to get default realm name: %s\n",
```

Comment:
I'm not sure if it is a good idea to remove the debug messages with some specific text. When reading the logs it might help to better understand which call failed. I'm not sure if the messages from `sss_krb5_get_error_message()` are always specific enough to understand which call failed.

So maybe you can keep something like `DEBUG(SSSDBG_OP_FAILURE, "Failed to get default realm name: %d\n", krberr);` here and then print the krb5 error message at the end?

Reply:
Thanks. I will take care.

Reply:
I restored all the debug messages with specific text.